.changeset/short-olives-train.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/plugin-kubernetes-backend': patch
+'@backstage/plugin-kubernetes-common': patch
```

Comment:
I think both of these changes are `minor` changes looking at the `api-reports` for both packages.